### DISCUSSION TITLE
Disable auto-slugify

### DISCUSSION THREAD

**Discussion by geff21st** [area17/twill#983] [Q&A] [answered]
Title and slug are connected now. Every time when title is edited, slug automatically changes with slugified copy of title. I'm looking for a way to disable this behaviour.  
Is there an option for this? I looked up through sources and didn't fing something similar.

**Comment by pboivin** [comment, marked as answer]
Hey @geff21st,

If you want to "unlink" the title and permalink fields, you can override [your module's `create` partial](https://github.com/area17/twill/blob/2.x/views/partials/create.blade.php) and comment out the `onChange` option :


**resources/views/admin/mymodule/create.blade.php**

```
@formField('input', [
    'name' => $titleFormKey ?? 'title',
    'label' => $titleFormKey === 'title' ? twillTrans('twill::lang.modal.title-field') : ucfirst($titleFormKey),
    'translated' => $translateTitle ?? false,
    'required' => true,
    // 'onChange' => 'formatPermalink'
])

@if ($permalink ?? true)
    @formField('input', [
        'name' => 'slug',
        'label' => twillTrans('twill::lang.modal.permalink-field'),
        'translated' => true,
        'ref' => 'permalink',
        'prefix' => $permalinkPrefix ?? ''
    ])
@endif
```

This will unlink the fields when creating new records and when updating existing records. You may want to keep the fields linked on create but not on update, to allow users to change the titles without automatically changing the slugs. The easiest way that comes to mind is to check the request to toggle the `onChange` option :

```
    'onChange' => request()->is('*/edit') ? '' : 'formatPermalink'
```

Hope this can help!

**Reply by geff21st** [reply]
thanks! it was really helpful
